test/SpecHelper.hs

```diff
@@ -184,4 +184,6 @@ createJsonData = do
       [H.stmt|
         insert into "1".json (data) values (?)
       |]
-      (J.object [("foo", J.object [("bar", J.String "baz")])])
+      (J.object [("id", J.Number 1)
```

Comment:
I wonder if you could use the json quasi-quoter here make this easier to read, like

```haskell
[H.stmt|
  insert into "1".json (data) values (?)
|] [json|
  [{"data": {"id": 1, "foo": {"bar": "baz"}}}]
|]
```